src/learning-catalogue/model/factory/moduleFactory.ts

```diff
@@ -8,11 +7,9 @@ import {ELearningModule} from '../eLearningModule'
 import {Module} from '../module'
 
 export class ModuleFactory {
-	private audienceFactory: AudienceFactory
 	private eventFactory: EventFactory
 
-	constructor(audienceFactory: AudienceFactory, eventFactory: EventFactory) {
-		this.audienceFactory = audienceFactory
+	constructor(eventFactory: EventFactory = new EventFactory()) {
```

Comment:
INFO - Why have we killed off the DI now?

Reply:
the way factories have been used was to call them with `new AnotherFactory()` if they required `AnotherFactory`, occasionally when a test double was required for tests it would have been passed on. So using a default parameter seems like a good solution - less lines of code and the same flexibility.

Reply:
ok